packages/components/src/components/wallet/small/small.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../../icon';
+import './small.scss';
+
+type TWalletSmallProps = {
+    icon?: string;
+    className?: string;
+    bg?: string;
+    data_testid?: string;
+    icon_class?: string;
+    size?: string;
+};
+
+const WalletSmall = ({ bg, icon, className, data_testid, icon_class, size = '16px' }: TWalletSmallProps) => {
+    if (!icon) {
+        return null;
+    }
+
+    return (
+        <div className={classNames('wallet-small', bg, className)}>
+            <Icon data_testid={data_testid} className={classNames(icon_class)} icon={icon} size={size} />
```

Comment:
I have a doubt here, as far as I remember the <Icon /> component can not show gradient background and the background will be white. please check this